### PR TITLE
[Minor] Fix typos in Ansor

### PR DESCRIPTION
**Pull request by junrushao1994**
Recently I installed a spell checker in VSCode...

### CODE REVIEW

**Review by comaniac — APPROVED** [review]

**Review by electriclilies — APPROVED** [review]
LGTM!

**Review by jcf94 — APPROVED** [review]
Thanks! LGTM. 😄

**Review by tqchen — APPROVED** [review]

**Comment by junrushao1994** [comment]
Let's wait for a few days before merging this PR, as I found some typos leftover

File: src/auto_scheduler/search_policy/utils.h [hunk 1]
```diff
@@ -645,11 +645,11 @@ class SplitFactorizationMemo {
...
 };
 
-/*! \brief Get the indexes of SplitStep that processes on spatial iteratior. */
+/*! \brief Get the indexes of SplitStep that processes on spatial iterator. */
 Array<Integer> GetSpatialSplitStepIds(const State& s, int stage_id);
 
 // Apply multi-level tiling structure according to a string format,
-// where "S" stands a space level, "R" stands for a reudciton level.
+// where "S" stands a space level, "R" stands for a reduction level.
 // For example, if the format is "SSRSRS", the we will
```
Comment (leandron) [inline src/auto_scheduler/search_policy/utils.h:653]:
```suggestion
// For example, if the format is "SSRSRS", then we will
```

File: include/tvm/auto_scheduler/compute_dag.h [hunk 2]
```diff
@@ -99,7 +99,7 @@ class AccessAnalyzer : public ObjectRef {
 
   /*!
    * \brief Return whether this operation is strictly inlinable
```
Comment (leandron) [inline include/tvm/auto_scheduler/compute_dag.h:101]:
```suggestion
   * \brief Return whether this operation is strictly inlineable
```

**Review by leandron — COMMENTED** [review]
Thanks for cleaning this up. A couple more for your collection.

**Comment by tqchen** [comment]
@junrushao1994 please incorporate @leandron 's comment then we can merge it in

**Comment by jroesch** [comment]
should wait for #6448 before landing

**Comment by tqchen** [comment]
@junrushao1994 please rebase and address @leandron 's comment

**Comment by junrushao1994** [comment]
Should be good to go :-)

**Comment by junrushao1994** [comment]
@tqchen @jcf94  Could you take another look? Thanks!

**Comment by tqchen** [comment]
Thanks @junrushao1994 @leandron  @jcf94 @electriclilies !

**Pull request merged**